### ISSUE TITLE
Creating new customer requisitions always shows a program window

### ISSUE THREAD

**Issue by bijaySussol** [openmsupply/mobile#3199]
## Describe the bug

Creating new customer requisitions always shows a program window

### To reproduce

1. Choose a non programme datafile
2. Sync to tablet
3. Tap to create a 'new customer requisitions'

### Expected behaviour

Only @joshxg knows
For me - If this is not a programme setting datafile - the program dialogue should be bypassed !

### Proposed Solution

Leave if you don't know how to fix/implement. Edit this issue description and explain here if you know the best path of implementing the fix within the codebase.

### Version and device info

- App version: mSupplyMobile-6_0_0-rc9-universal-release.apk
- Tablet model: SM-T515
- OS version: Android 9

### Additional context

@joshxg I do not know the idea here - so, thought to create a issue so that we know what is the idea with this customer requisition programe ordering

**Comment by joshxg** [comment]
I agree it is confusing! But: That is the requirement that craig DROWN gave so.. it is implemented..!

**Comment by katherine-sussol** [comment]
Re-opening as this seems to have come up numerous times in various discussions and caused much confusion 😆 . Discussed with @anildahalsussol and we will implement one of these options to reduce the confusion:
- Disable the New Requisition button if there are no programs available (less intrusive but less obvious _why_ they can't do it)
- Let the user press the button but instead of displaying the program modal, let the user know that they are unable to create a customer requisition without a program (a bit in your face but less room for confusion)
